### ISSUE TITLE
Podcast episodes 'pending'

### ISSUE THREAD

**Issue by rsinfotech** [ampache/ampache#3315]
Hi all, just noticed while trying to play a recent podcast that they don't seem to be being downloaded any longer?

All podcasts since Tuesday (09/08/22) are listed as 'pending' instead of 'complete' and obviously won't play in the web player:

![image](https://user-images.githubusercontent.com/3216809/184474358-6c2edd59-3e32-49e1-9898-6bd5b755b8d4.png)

The ampache logs reveal the following daily when the cron update of podcasts runs:

```
2022-08-12T22:32:30+01:00 [System] (Ampache\Repository\Model\Podcast) -> Episode source URL already exists, skipped
2022-08-12T22:32:30+01:00 [System] (Ampache\Repository\Model\Podcast) -> Episode source URL already exists, skipped
2022-08-12T22:32:31+01:00 [System] (Ampache\Repository\Model\Podcast_Episode) -> Downloading http://open.live.bbc.co.uk/mediaselector/6/redir/version/2.0/mediaset/audio-nondrm-download/proto/http/vpid/p0cssrlv.mp3 to /mnt/ampachemusic/Podcasts2020/Six O'Clock News/1660325940-12_08_2022 Salman Rushdie Stabbed on Stage-p0cssrlv.mp3 ...
2022-08-12T22:32:38+01:00 [System] (Ampache\Repository\Model\Podcast_Episode) -> Download completed.
2022-08-12T22:32:38+01:00 [System] (Ampache\Repository\Model\Podcast_Episode) -> Updating details /mnt/ampachemusic/Podcasts2020/Six O'Clock News/1660325940-12_08_2022 Salman Rushdie Stabbed on Stage-p0cssrlv.mp3 ...
2022-08-12T22:32:38+01:00 [System] (Ampache\Repository\Model\Catalog) -> update_media_from_tags: Error loading file 
2022-08-12T22:32:38+01:00 [System] (Ampache\Repository\Model\Podcast_Episode) -> Updating details /mnt/ampachemusic/Podcasts2020/Six O'Clock News/1660239780-11_08_2022 Dry Weather Leads to Drought, Fire Warnings-p0csksq8.mp3 ...
2022-08-12T22:32:38+01:00 [System] (Ampache\Repository\Model\Catalog) -> update_media_from_tags: Error loading file 
2022-08-12T22:32:38+01:00 [System] (Ampache\Repository\Model\Podcast_Episode) -> Updating details /mnt/ampachemusic/Podcasts2020/Six O'Clock News/1660153680-10_08_2022 Mounting Calls for Cost of Living Help-p0cs8prr.mp3 ...
2022-08-12T22:32:38+01:00 [System] (Ampache\Repository\Model\Catalog) -> update_media_from_tags: Error loading file 
2022-08-12T22:32:38+01:00 [System] (Ampache\Repository\Model\Podcast_Episode) -> Updating details /mnt/ampachemusic/Podcasts2020/Six O'Clock News/1660067100-09_08_2022 Energy Bills May Top £4,000-p0cs1mbr.mp3 ...
2022-08-12T22:32:38+01:00 [System] (Ampache\Repository\Model\Catalog) -> update_media_from_tags: Error loading file 
2022-08-12T22:32:38+01:00 [System] (Ampache\Repository\Model\Podcast) -> Syncing feed https://podcasts.files.bbci.co.uk/p02pc9qx.rss ...
2022-08-12T22:32:38+01:00 [System] (Ampache\Repository\Model\Podcast) -> Adding new episode to podcast 39... 1660276800
2022-08-12T22:32:38+01:00 [System] (Ampache\Repository\Model\Podcast) -> Episode source URL already exists, skipped
```
I'm on the latest develop commit and as mentioned, this seems to have been happening since Tuesday (09/08/22) if this helps?

**Comment by rsinfotech** [comment]
Sorry, this is a misleading description, the podcasts are being downloaded to the expected directory and are all playable directly from there.

 ``(Ampache\Repository\Model\Catalog) -> update_media_from_tags: Error loading file`` seems to be the problem. 

All the files that error when loaded by ampache are listed as 'pending' instead of 'completed' in the web client so just can't be played from there, they have been downloaded by the cron job and are all valid, playable files.

With nobody else having this issue, I'm just checking things at my end for any other possible reason these suddenly became unreadable by ampache on Tuesday?

**Comment by rsinfotech** [comment]
Reverting to commit f83eb4683 of 08/08/22 cures this for me. 

Ran the cron command and all podcasts from Tuesday onwards got downloaded again and ampache didn't complain about loading the files and all now listed as completed in the web interface and fully playable. :+1:

Notice lots of changes to podcasts since then on more nice features such as the search facility but something here is breaking it on my setup?

**Comment by lachlan-00** [comment]
i'm checking now, will add a new link and see what happens too. nothing obvious just browsing though

**Comment by lachlan-00** [comment]
fixed with da4fc8592

the function used to set the $media->file attribute but i took that out so the thing would fail.

now it will update the database and the tags on pending.
